Modules/Filtering/DisplacementField/include/itkDisplacementFieldTransform.h

```diff
@@ -181,7 +183,9 @@ class ITK_TEMPLATE_EXPORT DisplacementFieldTransform : public Transform<TParamet
    * container. */
   virtual void
   SetDisplacementField(DisplacementFieldType * field);
-  SetDisplacementField(VectorImage<TParametersValueType, VDimension> * field);
+  virtual void
+               SetDisplacementField(VectorImageDisplacementFieldType * field);
+  virtual void SetDisplacementField(nullptr);
```

Comment:
Did you meant `nullptr_t` instead of `nullptr`?

Reply:
Thanks @dzenanz , this will help make the interface much cleaner